llama_stack/cli/stack/_build.py

```diff
@@ -171,11 +173,22 @@ def run_stack_build_command(
                 )
                 return
 
-    _run_stack_build_command_from_build_config(build_config, image_name=image_name)
+        if build_config.image_type == ImageType.container.value and not args.image_name:
```

Comment:
@terrytangyuan your `llama stack build` commands will now need a `--image-name` option because the `BuildConfig` data structure no longer has a name.